fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalTableFunctionOperator.java

```diff
@@ -22,10 +23,11 @@ public class LogicalTableFunctionOperator extends LogicalOperator {
     //External column ref of the join logic generated by the table function
     private ColumnRefSet outerColumnRefSet;
     //table function input parameters
-    private final Map<ColumnRefOperator, ScalarOperator> fnParamColumnProjectMap;
+    //private final Map<ColumnRefOperator, ScalarOperator> fnParamColumnProjectMap;
+    private final List<Pair<ColumnRefOperator, ScalarOperator>> fnParamColumnProjectMap;
```

Comment:
To allow duplicated param?

Reply:
fnParamColumnProjectMap is an input parameter for recording functions and cannot be deleted. It turned out that Map was used, which caused the same key to be deduplicated, and the output column and input column could not correspond. This time, it caused BE crash

Reply:
Can you explain it with a simple example and add the example in the pr description?

Reply:
For example, in this sql "select * from tarray, unnest(v3, v3)", in the problematic version, the keyList of getFnParamColumnProject is "[3]", because a Map is used, which causes the key of FnParam to be deduplicated. And this parameter represents the input param of the table function, which cannot be deduplicated, which will lead to the inability to correctly obtain the input parameter column when the function is executed.